utils/sql/db_update_manifest.txt

```diff
@@ -471,6 +471,7 @@
 9215|2023_01_08_zone_max_level.sql|SHOW COLUMNS FROM `zone` LIKE 'max_level'|empty|
 9216|2023_01_15_merc_data.sql|SHOW TABLES LIKE 'mercs'|empty|
 9217|2023_01_15_chatchannel_reserved_names.sql|SHOW TABLES LIKE 'chatchannel_reserved_names'|empty|
+9218|2023_01_24_item_recast.sql|SELECT * FROM db_version WHERE version >= 9218|empty|
```

Comment:
Can we change this to be more specific? We try not to check against arbitrary versions since it's not idempotent in this case.

Suggest:

Look for the condition of the previous state to trigger running the desired state. Character recast currently shows as a `smallint` column

```
9218|2023_01_24_item_recast.sql|show columns from character_item_recast like '%recast_type%'|contains|smallint
```